### PR TITLE
Fix/#296 cleaning table data new

### PR DESCRIPTION
**Pull request by AnyaHe**
Cherry-picked changes of branch #301 
Test function in original branch shows varying behaviour for different versions of python. Needs to be fixed as a seperate issue. Therefore only changes concerning the cleaning of table data are adapted in this branch.

### CODE REVIEW

**Comment by AnyaHe** [comment]
Please check together with [ding0](https://github.com/openego/eDisGo/pull/154/files/8c8686456d8e096112ffe6188900c3390da8709b)

**Comment by birgits** [comment]
Thank you for your fixes! One thing I noticed is that you also changed delimiters to ';'. As comma is the default I would not change it.

File: ding0/tools/pypsa_io.py [hunk 1]
```diff
@@ -329,17 +330,17 @@ def edges_to_dict_of_dataframes(grid, edges):
                               str(edge['branch'].id_db)])
 
         # TODO: find the real cause for being L, C, I_th_max type of Series
-        if (isinstance(edge['branch'].type['L'], Series) or
-                isinstance(edge['branch'].type['C'], Series)):
-            x = omega * edge['branch'].type['L'].values[0] * 1e-3
+        if (isinstance(edge['branch'].type['L_l'], Series) or#warum wird hier c abgefragt?
```
Comment (birgits) [inline ding0/tools/pypsa_io.py]: Should we clarify why C is checked as well?

**Comment by AnyaHe** [comment]
> 
> 
> Thank you for your fixes! One thing I noticed is that you also changed delimiters to ';'. As comma is the default I would not change it. Further, I would use selfspeaking names such as R_per_km rather than R_l.

Thank you for the comments! I noticed, that in model_grids-lv_string_properties.csv and model_grids-lv_relation_apartment_string.csv the delimiter is ';' as well and the decimal sign ','. Should I change that as well to be consistant or is it not as important?

**Comment by birgits** [comment]
That would make sense, if it's not too much effort. Thanks!

**Comment by AnyaHe** [comment]
I added the amendments you requested. I hope the branch is ready to merge now.

**Pull request merged**